### ISSUE TITLE
Attach to parent element

### ISSUE THREAD

**Issue by asacarter** [taufik-nurrohman/color-picker#75]
Was the `parent` option to attach the picker to a parent element removed in V2?

The problem is that if it's attached to a scrollable element, the popup is positioned to window so it doesn't scroll with the scrollable element.

**Comment by taufik-nurrohman** [comment]
Yes. See the [static color picker example](https://taufik-nurrohman.js.org/color-picker/tweak/static.html) on how I append the picker element to a DOM:

~~~ js
picker.enter(document.querySelector('#container'));
~~~
